### DISCUSSION TITLE
Tailwind Classes not getting applied by javascript

### DISCUSSION THREAD

**Discussion by itspriyambhattacharya** [tailwindlabs/tailwindcss#7633] [Help]
When I'm using tailwind classes like pointer-events-none in html , it is working properly . But when i want to add the same class by js (main.classList.toggle('pointer-events-none')), it is not working.
Here pe-none is not a tailwind class. I have to create it manually.
Please Help.
![doubt](https://user-images.githubusercontent.com/84953701/155476117-e8339c6e-8eac-4aaf-9276-03e7c700e7b4.png)

**Comment by ExEr7um** [comment]
Can you share your `tailwind.config.js` file? Maybe you have wrongly configured `content` property.

**Reply by adamwathan** [reply]
+1, very likely that the problem is that your `content` configuration isn't covering the `app.js` file.

**Reply by itspriyambhattacharya** [reply]
I have tried adding app.js in content but my site breaks after adding it.
This is my current tailwind.config.js
![tailwind](https://user-images.githubusercontent.com/84953701/155654991-b588b3bf-5951-4d11-af5e-b282c821c9c0.png)

**Reply by simonswiss** [reply]
Hey!

Like @ExEr7um and Adam have mentioned, since you're adding Tailwind classes from a `.js` file, you need to specify the path to that file (or multiple files) in the `content` array, since currently it's only looking at `.html` files.

Your `content` option should look something like that (but make sure all the files where you use Tailwind classes are referenced in that array:

```diff
content: [
  "./**/*.html",
+  "./app.js",
+  // ...
],
```

If your site breaks, you probably have a syntax error in the way you added that `app.js` file in the array. All this does is tell the JIT engine in Tailwind to look at more files - it should not break your site in any way since all it will do is potentially add more styles to your CSS output.
